ArduPlane/Parameters.h

```diff
@@ -51,7 +51,7 @@ class Parameters {
 
         // Misc
         //
-        k_param_auto_trim      = 10,
+        k_param_auto_trim      = 10, // unused
```

Comment:
```suggestion
        k_param_auto_trim_unused      = 10, // unused
```

Reply:
There are lots without this, including the ones I recently removed....